panic: runtime error: index out of range [0] with length 0

Mac
When I try to run the anyflip downloader file with terminal, this happens before I can enter any commands, unprompted and immediately upon opening the program:


`Device_Name:~ Username$ /Users/Username/.local/bin/anyflip-downloader ; exit;
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.run()
	/home/runner/work/anyflip-downloader/anyflip-downloader/main.go:61 +0x3a5
main.main()
	/home/runner/work/anyflip-downloader/anyflip-downloader/main.go:57 +0x52
logout

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.


[Process completed]

`

--- a/main.go
+++ b/main.go
@@ -58,6 +58,12 @@ func main() {
 }
 
 func run() {
+	if flag.NArg() == 0 {
+		fmt.Fprintln(os.Stderr, "error: no URL provided")
+		printUsage()
+		os.Exit(1)
+	}
+
 	anyflipURL, err := url.Parse(flag.Args()[0])
 	if err != nil {
 		log.Fatal(err)

